CHANGELOG.md

```diff
@@ -3,10 +3,13 @@
 ## Next release
 
 ## Features Added
+- Build and docker image to use Java 17
 - Updated Docker image to use the latest Ubuntu LTS image
 
 ### Bugs Fixed
 - Update grpc to version 1.57.2 to fix CVE-2023-33953
+- Update Azure libraries to fix CVE-2023-36415
+- Update okhttp logging interceptor version to fix CVE-2023-0833
```

Comment:
Do we need to mention web3j in here too?

Reply:
we might mention web3j in "features added" ? as we are overriding logging-interceptor anyways ...

Reply:
done